controlpanel/frontend/static/javascripts/modules/tool-status.js

```diff
@@ -34,27 +34,28 @@ moj.Modules.toolStatus = {
   },
 
   buildEventHandler(listener) {
+    const toolstatus = this;
```

Comment:
`this` seems silly.

Reply:
`this` in JavaScript is problematic, a source of many bugs (like the one fixed in the latest commit) and requires non-trivial knowledge of how JavaScript scopes things in order to spot where such things get FUBAR.